src/components/FileRepo/AggregationSidebar/AggregationSidebar.js

```diff
@@ -35,21 +36,41 @@ const AggregationWrapper = styled(Column)`
 
 const AggregationHeader = styled('div')`
   display: flex;
-  padding: 15px 7px 15px 12px;
   align-items: center;
+  padding: 15px 7px 15px 12px;
 `;
 
-const AggregationTitle = styled(Heading)`
-  flex-grow: 1;
-  margin-bottom: 0px;
-  font-size: 18px;
+const OpenActionChevron = styled(RightChevron)`
+  margin-top: 10px;
+  min-width: 14px;
 `;
 
-const Controls = styled(Column)`
-  flex: 0 0 auto;
+const CloseActionChevron = styled(LeftChevron)`
+  margin-left: auto;
 `;
 
-const AggregationSidebar = compose(injectState, withTheme, withApi)(
+const BrowseAllButton = styled(TealActionButton)`
+  margin-left: 10px;
+`;
+
+const CollapsibleAggregationWrapper = styled(AggregationWrapper)`
+  transition: all 0.25s;
+  min-width: 37px;
+  flex: inherit;
+  width: ${({ expanded }) => (expanded ? `100%` : `0%`)} !important;
+  overflow: hidden;
+`;
+
+const CollapsibleCustomAggregationsPanel = styled(CustomAggregationsPanel)`
+  display: ${({ expanded }) => (expanded ? `block` : `none`)};
+`;
```

Comment:
`CustomAggregationsPanel` will not work with `styled` because it doesn't have a `className` prop that's forwarded to a DOM element, I think what you are looking to do is a `styled('div')` and using it to wrap `CustomAggregationsPanel` in jsx. Might need a `height:100%` as well as it's a new dom element